roles/openshift_management/defaults/main.yml

```diff
@@ -5,6 +5,8 @@ openshift_management_project: openshift-management
 openshift_management_project_description: CloudForms Management Engine
 # Number of retries when waiting for the app to start (retried every 30 seconds)
 openshift_management_pod_rollout_retries: 30
+# Node selector for project:
+openshift_management_nodeselector: region=infra
```

Comment:
Should be `node-role.kubernetes.io/infra=true`

Reply:
I assumed it was a var, so I put in quotes, should be fine now?

Reply:
Changed it, should be fine now.